boards/weact/usb2canfdv1/doc/index.rst

```diff
@@ -0,0 +1,65 @@
+.. _usb2canfdv1:
+
+WeAct Studio USB2CANFDV1
+#########################
+
```

Comment:
```suggestion
.. zephyr:board:: usb2canfdv1
```

Reply:
Changed to use the `zephyr:board::` directive.